dev-support/Jenkinsfile

```diff
@@ -131,9 +131,11 @@ curl -L  -o personality.sh "${env.PROJECT_PERSONALITY}"
             // TODO does hadoopcheck need to be jdk specific?
             // Should be things that work with multijdk
             TESTS = 'all,-unit,-findbugs'
-            // on branches that don't support jdk7, this will already be JAVA_HOME, so we'll end up not
-            // doing multijdk there.
-            MULTIJDK = '/usr/lib/jvm/java-8-openjdk-amd64'
+            // all jdks tested on this stage for this branch
+            // note that JAVA_HOME still needs to be set for tests
+            // that don't support multijdk. JAVA_HOME will also
+            // determine which jvm will be tested last.
+            MULTIJDK = '/usr/lib/jvm/zulu-8-amd64,/usr/lib/jvm/zulu-7-amd64'
```

Comment:
nit: I expected to see these ordered ascending by version number.

Reply:
if it weren't for the JAVA_HOME handling, they'd be run in this order. due to the way jdk compatibility rules work having this in descending makes more sense to me. because if we run a sequence of tests that needed to use the output of a prior test, then stopping at an older JDK and starting at a new JDK would work, but not the reverse.